app/webpacker/components/RegistrationsV2/Register/RegistrationClosedMessage.jsx

```diff
@@ -0,0 +1,48 @@
+import { DateTime } from 'luxon';
+import React, { useEffect, useState } from 'react';
+import { Message } from 'semantic-ui-react';
+import I18nHTMLTranslate from '../../I18nHTMLTranslate';
+
+export default function RegistrationClosedMessage({
+  registrationStart,
+  onTimerEnd,
+}) {
+  const [timeLeft, setTimeLeft] = useState({
+    days: 0, hours: 0, minutes: 0, seconds: 0,
+  });
+
+  const start = DateTime.fromISO(registrationStart);
+
+  // Update the timer every second
+  useEffect(() => {
+    const updateTimer = () => {
+      const now = DateTime.local();
+
+      if (now >= start) {
+        onTimerEnd();
+        return;
+      }
+
+      const diff = start.diff(now, ['days', 'hours', 'minutes', 'seconds']).toObject();
+      setTimeLeft({
+        days: Math.floor(diff.days),
+        hours: Math.floor(diff.hours),
+        minutes: Math.floor(diff.minutes),
+        seconds: Math.floor(diff.seconds),
+      });
+    };
+
+    const intervalId = setInterval(updateTimer, 1000); // Update every second
```

Comment:
I'm not sure how important this is for performance or anything, but if there's more than 1 hour left then you could set the next update to happen when there's an hour left. Although then the `days`, `hours`, `minutes`, and `seconds` values will be incorrect, so maybe this is a bad suggestion.

Reply:
mh.. I don't really expect anyone to sit on the page if it's more than an hour away

Reply:
I guess `const [timeLeft, setTimeLeft] = useState(start.diff(DateTime.local(), ['days', 'hours', 'minutes', 'seconds']).toObject());` this would be enough though